collection/lens.rkt

```diff
@@ -39,6 +67,21 @@
   (make-lens (λ (target) (nth target index))
              (λ (target value) (set-nth target index value))))
 
+(define (mapper-lens lens)
```

Comment:
That's one of the reasons I put `mapper` instead of `map`, because `map` sounds like a verb, but `mapper` sounds like the noun version.